src/genie/libs/parser/ios/asr901/tests/ShowEnvironment/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,72 @@
+expected_output = {
+    "Fan": {
+        "fan1": {
+            "speed": "53",
+            "status": "Normal"
+        },
+        "fan2": {
+            "speed": "53",
+            "status": "Normal"
+        },
+        "fan3": {
+            "speed": "54",
+            "status": "Normal"
+        }
+    },
+    "PowerSupply": {
+        "0.75V": {
+            "status": "Normal",
+            "volt": "+0.760"
+        },
```

Comment:
I would suggest to have `volt` as float
```suggestion
    "power_suppy": {
        "0.75V": {
            "status": "Normal",
            "volt": 0.760 # float
        },
```